### ISSUE TITLE
can it run with CPU only?

### ISSUE THREAD

**Issue by Darienxccs4** [rimsila/fooocus-API-pinokio#1]
Is there an option to run Fooocus on Pinokio using CPU only?

**Comment by rimsila** [comment]
@Darienxccs4 
> Is there an option to run Fooocus on Pinokio using CPU only?

I have tried only with Nvidia GPU but it's seem has option to use CPU too  you try it first
![image](https://github.com/rimsila/fooocus-API-pinokio/assets/51617383/f6895b3b-32a3-4098-8402-dfc55dc354f3)

Also base on Fooocus Benchmark using CPU it's very slow
![image](https://github.com/rimsila/fooocus-API-pinokio/assets/51617383/ae50144d-469c-4c52-9255-b216efba18b0)
[https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement](https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement)

**Comment by Darienxccs4** [comment]
Yes i had read that it is slow but i dont have Nvidia GPU, i also read a way to set it up in the "run.bat" but that's when you download it directly from Github, but on Pinokio there isn't an "run.bat", so that's my question, does anybody knows how to set it up in Pinokio?

**Comment by rimsila** [comment]
> Yes i had read that it is slow but i dont have Nvidia GPU, i also read a way to set it up in the "run.bat" but that's when you download it directly from Github, but on Pinokio there isn't an "run.bat", so that's my question, does anybody knows how to set it up in Pinokio?

When you open Pinokio app after downloaded and open Fooocus API app then you can click left menu -> install after finished setting up -> run it will open Like this https://github.com/rimsila/fooocus-API-pinokio/blob/main/image.png
